src/main/jbake/content/blog/2017/01/13/events-aggregation-extension.adoc

```diff
@@ -0,0 +1,167 @@
+= Extending Complex Event Processing in Hawkular Alerting
+Lucas Ponce
+2017-01-13
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerts, complex, events, processing, stream, polling
+
+== Stream Processing versus Polling Processing
+
+Hawkular Alerting uses different techniques to detect behaviours by defining rules.
+
+The main implementation is based in a Stream Processing design. +
+Hawkular Alerting analyzes streams of data and events ordered in a timeline searching matches against conditions. +
+This technique is also known as Event Processing (or Complex Event Processing)specially when the processing of incoming data might need multiple conditions.
+
+This method is particularity good to identify meaningful happenings on complex scenarios and respond to them as quickly as possible.
+
+Hawkular Alerting also supports Polling techniques based on Alerters that allow to define queries that are processing
```

Comment:
"supports a polling technique"
"allow us to define"
"processed" instead of "processing"?
"periodic" instead of "periodically"